front-end/src/components/newsfeed-view/news-list/card-group/news-card/FacebookCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Timestamp from './elements/Timestamp';
+import styles from './social-media-card.module.scss';
+import ExpandableContent from './elements/ExpandableContent';
+
+const FacebookCard = ({data}) => (
+    <div className={`${styles['social-media-card']} ${styles['facebook']}`}>
+        <div className={styles['service-source']}>
+            {data.source}
+        </div>
+        <div className={styles['content']}>
+            <div className={styles['content-source']}>
+                <img src={data.thumbnail} alt='Facebook thumbnail' />
+                <div>{data.author}</div>
+            </div>
+            <div className={styles['description']}>
+                <ExpandableContent 
+                    maxLine={4} 
+                    description={data.description} 
+                    ellipsis='... more'
```

Comment:
do we have a finnish alternative to 'more' and is it displayed fine? Or will we have 'more' for all language settings? Should ask Laura probably

Reply:
Good question. Will get it clarified by Laura or Daniella.